compiler/env/OMRClassEnv.hpp

```diff
@@ -141,6 +143,16 @@ class OMR_EXTENSIBLE ClassEnv
     */
    const TR::TypeLayout* enumerateFields(TR::Region& region, TR_OpaqueClassBlock * clazz, TR::Compilation *comp) { return NULL; }
 
+   /**
+    * @brief Determines if a class hass less than two concrete classes in the
```

Comment:
"Determine if a list of classes contains less than two concrete classes. 
 A class is considered concrete if it is not an interface or an abstract class"